ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs

```diff
@@ -10,7 +10,10 @@ namespace ReactNative.UIManager
     /// </summary>
     public class LayoutShadowNode : ReactShadowNode
     {
-        private const float Undefined = CSSConstants.Undefined;
+        /// <summary>
+        /// Undefined float value.
+        /// </summary>
+        protected internal const float Undefined = CSSConstants.Undefined;
```

Comment:
Any reason we can't delete this constant all together and just inline the reference to CSSConstants.Undefined?